library-best-practices/JLBP-4.md

```diff
@@ -1,17 +1,17 @@
 [JLBP-4] Avoid dependencies on unstable libraries and features
 --------------------------------------------------------------
 
-- Unstable libraries are libraries that allow for breaking changes to their
-  Public API within the same major version. For libraries following semver, this
-  means libraries with a 0.x.y version. (See [JLBP-3](JLBP-3.md) for more
-  details on the recommendations for following semver.)
-- Unstable features are features that are not part of the stable Public API of a
-  stable library, often marked with annotations like `@Beta`.  (See
+- Unstable libraries are libraries that allow breaking changes to their
+  public API within the same major version. For libraries following semantic
+  versioning, this means libraries with a 0.x.y version. (See [JLBP-3](JLBP-3.md) 
+  for more details on the recommendations for semantic versioning.)
+- Unstable features are not part of the public API of a
```

Comment:
Removing "are features that" changes the meaning here. 

- Prior wording: the "given" is features that aren't part of the stable public API; this definition gives those a label.
- New wording: the "given" is unstable features, and allows for other features that are not part of the public API but aren't called "unstable."

I think this rewording makes it less clear.

Reply:
Reworded. let me know what you think now.